resolwe/flow/docker_images/Dockerfile.communication

```diff
@@ -8,7 +8,8 @@ RUN apk add --update --no-cache build-base libffi-dev zeromq-dev openssl-dev
 RUN pip install --upgrade pip
 # Keep the versions bellow in sync with versions in setup.py.
 RUN pip install --prefix=/install --no-warn-script-location \
-    wrapt~=1.11.1 crcmod google-cloud-storage~=1.35.0 boto3~=1.16.47 pyzmq~=20.0.0
+    wrapt~=1.11.1 google-cloud-storage~=1.35.0 boto3~=1.16.47 pyzmq~=20.0.0 \
+    google-crc32c==1.1.2 crcmod 
```

Comment:
Put the pinned dependency in a separate line and add a comment on why this is required.